src/components/TodoContainer.js

```diff
@@ -1,12 +1,97 @@
-import React from "react"
-class TodoContainer extends React.Component {
-  render() {
-    return (
-      <div>
-        <h1>Hello from Create React App</h1>
-        <p>I am in a React Component!</p>
-      </div>
-    )
+/* eslint-disable */
+import React, { useState, useEffect } from 'react';
+import { Route, Routes, BrowserRouter as Router } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+import About from '../pages/About';
+import NotMatch from '../pages/NotMatch';
+import Navbar from './Navbar';
+
+const TodoContainer = () => {
+  const [todos, setTodos] = useState(getInitialTodos());
+  // useEffect(() => {
+  //   // getting stored items
+  //   const temp = localStorage.getItem("todos")
+  //   const loadedTodos = JSON.parse(temp)
```

Comment:
Kindly keep in mind that commented code should never be pushed remotely. Please remove all commented/ unnecessary code before pushing your changes, to make your work look more presentable and clean

Reply:
@Ellon-M 
I have removed the images that I don't need on this [commit](https://github.com/Lucash2022/react-todo-app/pull/2/commits/820365960f148e148b774577e7a103e9e55c59b8)
The ones that remain are used in the README file
![image](https://user-images.githubusercontent.com/41428579/212851643-07803ed2-3831-4a8a-babc-460a0b728f0b.png)